tools/travis/splits/split_misc.sh

```diff
@@ -76,5 +76,7 @@ run_test "SQL Client end-to-end test for modern Kafka" "$END_TO_END_DIR/test-scr
 
 run_test "Dependency shading of table modules test" "$END_TO_END_DIR/test-scripts/test_table_shaded_dependencies.sh"
 
+run_test "Shaded Hadoop S3A with credentials provider end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh hadoop_with_provider"
```

Comment:
Unrelated to the changes in this PR, but could you also move:
```
run_test "Shaded Hadoop S3A end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh hadoop"
run_test "Shaded Presto S3 end-to-end test" "$END_TO_END_DIR/test-scripts/test_batch_wordcount.sh presto"
```
from `run-pre-commit-tests.sh` here, so they are aggregated in one place?

Apparently, the above tests are not actually running during pre-commit tests run:
```
==============================================================================
Running 'Shaded Hadoop S3A end-to-end test'
==============================================================================
...
Did not find AWS environment variables, NOT running the e2e test.
...
[PASS] 'Shaded Hadoop S3A end-to-end test' passed after 0 minutes and 0 seconds! Test exited with exit code 0.

==============================================================================
Running 'Shaded Presto S3 end-to-end test'
==============================================================================
...
Did not find AWS environment variables, NOT running the e2e test.
...
[PASS] 'Shaded Presto S3 end-to-end test' passed after 0 minutes and 0 seconds! Test exited with exit code 0.
```

Reply:
The s3 e2e tests rely on environment variables that are only available on actual branches, not pull requests. On a branch (like `master`) these tests will be run on each commit.